Source/WebCore/platform/graphics/ca/PlatformCALayer.h

```diff
@@ -210,8 +210,8 @@ class WEBCORE_EXPORT PlatformCALayer : public ThreadSafeRefCounted<PlatformCALay
     virtual bool acceleratesDrawing() const = 0;
     virtual void setAcceleratesDrawing(bool) = 0;
 
-    virtual bool wantsDeepColorBackingStore() const = 0;
-    virtual void setWantsDeepColorBackingStore(bool) = 0;
+    virtual ContentsFormat contentsFormat() const = 0;
+    virtual void setContentsFormat(ContentsFormat) = 0;
```

Comment:
You may need to forward-declare `ContentsFormat` near the top of this file (unless it's somehow already known through an include?)

Reply:
The forward declaration is added.